test/unit/component/parser/json/JsonTestUtils.ts

```diff
@@ -166,26 +166,6 @@ export function verifyEdge(edge: Edge, expectedValue: ExpectedEdge | ExpectedSeq
   }
 }
 
-export function verifyEvent(model: BpmnModel, kind: ShapeBpmnEventKind, expectedNumber: number): void {
-  const events = model.flowNodes.filter(shape => {
-    const bpmnElement = shape.bpmnElement;
-    return bpmnElement instanceof ShapeBpmnEvent && (bpmnElement as ShapeBpmnEvent).eventKind === kind;
-  });
-  expect(events).toHaveLength(expectedNumber);
-}
-
-export function verifyBoundaryEvent(model: BpmnModel, kind: ShapeBpmnEventKind, expectedNumber: number, isInterrupting?: boolean): void {
```

Comment:
ok, this function was only needed for boundary event tests and is now located directly in the event tests file